src/init.cpp

```diff
@@ -627,6 +627,7 @@ void ThreadImport(std::vector<fs::path> vImportFiles)
 {
     const CChainParams& chainparams = Params();
     RenameThread("bitcoin-loadblk");
+    ScheduleBatchPriority();
```

Comment:
Would be nice to log the error message if sched_batch returns -1. EINVAL / EPERM cases would be good to know about.